SSA/Projects/InstCombine/ForMathlib.lean

```diff
@@ -366,7 +366,7 @@ def srem? {w : Nat} (x y : Bitvec w) : Option $ Bitvec w :=
 /--
  If the condition is an i1 and it evaluates to 1, the instruction returns the first value argument; otherwise, it returns the second value argument.
 -/
-def select {w : Nat} (c : Bitvec 1) (x y : Bitvec w) : Bitvec w :=
+protected def select {w : Nat} (c : Bitvec 1) (x y : Bitvec w) : Bitvec w :=
```

Comment:
why is `select` protected?

Reply:
Oh, good point, maybe I should have explained that in a comment or so.

Basically, for some reason I do not understand, `open Bitvec` does not bring `select` into scope in `AliveStatements.lean`, so by making it protected, the extraction mecanism via my hacky `print_goal_as_error` extracts it with the right namespace 😅 maybe it does make sense to revert this or even investigate why the `open` statement won't work...